docs/config/server-options.md

```diff
@@ -90,7 +90,7 @@ Configure custom proxy rules for the dev server. Expects an object of `{ key: op
 
 Note that if you are using non-relative [`base`](/config/shared-options.md#base), you must prefix each key with that `base`.
 
-Extends [`http-proxy`](https://github.com/http-party/node-http-proxy#options). Additional options are [here](https://github.com/vitejs/vite/blob/main/packages/vite/src/node/server/middlewares/proxy.ts#L13).
+Extends [`http-proxy`](https://github.com/http-party/node-http-proxy#options). Additional options are [here](https://github.com/vitejs/vite/blob/68bd0a84011eb2bd8a7bb03da6e95acc97e466ea/packages/vite/src/node/server/middlewares/proxy.ts#L12).
```

Comment:
I think we should keep pointing to `main`, so there are more chances the user will see the latest options even if we need to update the line when things change.

Reply:
Makes sense.